.github/workflows/documentation.yml

```
@@ -0,0 +1,134 @@
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/documentation.yml'
+      - 'doc/**'
+      - 'locale/**'
+      - 'CMakeLists.txt'
+
+    branches-ignore:
+      - 'gh-pages'
+
+    tags: []
```

actionlint (1.7.7)

"tags" section should not be empty

(syntax-check)